specification/_doc_ids/table.csv

```diff
@@ -522,6 +522,7 @@ point-in-time-api,https://www.elastic.co/docs/api/doc/elasticsearch/operation/op
 prevalidate-node-removal,https://www.elastic.co/docs/api/doc/elasticsearch/group/endpoint-cluster,https://www.elastic.co/guide/en/elasticsearch/reference/8.18/cluster.html,
 preview-dfanalytics,https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-ml-preview-data-frame-analytics,https://www.elastic.co/guide/en/elasticsearch/reference/8.18/preview-dfanalytics.html,
 preview-transform,https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-transform-preview-transform,https://www.elastic.co/guide/en/elasticsearch/reference/8.18/preview-transform.html,
+project-tags,todo,,
```

Comment:
Some downstream docs generation tools may try to inject this `todo` as a URL in an HTML link, which could cause CI failures when docs jobs run their link-checker. Maybe try leaving it empty for now?